# For customize messages handling, see doc at 
# http://ros.org/wiki/rtt_ros_integration

# Primitive .msg found at 
# http://ros.org/wiki/geometry_msgs

# Sensor position 
geometry_msgs/Point position

# Sensor orientation
geometry_msgs/Quaternion orientation

# Sensor error
float64 error

# Sensor visibility
bool visible

# Port for connection pc/Aurora
uint32 portHandle
